impl/src/context.rs

```diff
@@ -17,9 +17,9 @@ use serde::Serialize;
 
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize)]
 pub struct BuildableDependency {
+  pub buildable_target: String,
```

Comment:
What's up with these changes? The don't seem relevant to the PR (and, in this case, i'm not sure why you're making them).

Reply:
These are required to have the outputs be sorted correctly

Reply:
since Buildifier will expect the files to be sorted by `buildable_target` (the thing that gets renderd) instead of `name`. See #261 and https://github.com/google/cargo-raze/pull/260#issuecomment-710762137 for details about the issue.

Reply:
Oh I think I understand. To be clear, we're relying on rust sorting behavior for the resulting ordering of the generated BUILD rule. This is important to ensure that the output is BUILDIFIER compliant.

Perhaps capture this in struct documentation to prevent someone from changing this later without understanding why these fields are ordered in this way?

Reply:
I added documentation for the struct and a note about buildifier compliance